server/config/config.go

```diff
@@ -22,6 +22,8 @@ const (
 	DefaultGRPCWebAddress = "0.0.0.0:9091"
 )
 
+var ErrSetMinGasPrice = errors.New("please set min gas price in app.toml or flag or env var")
```

Comment:
@ruhatch Since we use the same error text in [ValidateBasic()](https://github.com/cosmos/cosmos-sdk/blob/cyberbono3/non-zero-default-fees/server/config/config.go#L313)  and in [TestEmptyMinGasPrices](https://github.com/cosmos/cosmos-sdk/blob/cyberbono3/non-zero-default-fees/server/util_test.go#L437), i have decided to use `ErrSetMinGasPrice`. Does it sound good to you?

Reply:
Can we add godocs to exported stuff?